modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -25,22 +25,33 @@ function perflab_aea_audit_enqueued_scripts() {
 		global $wp_scripts;
 		$enqueued_scripts = array();
 
-		foreach ( $wp_scripts->queue as $handle ) {
-			$src = $wp_scripts->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+		foreach ( $wp_scripts->done as $handle ) {
+			$script = $wp_scripts->registered[ $handle ];
+
+			if ( $script->src && ! strpos( $script->src, 'wp-includes' ) ) {
+
+				// Add any extra data (inlined) that was passed with the script.
+				$inline_size = 0;
+				if ( ! empty( $script->extra ) && ! empty( $script->extra['after'] ) ) {
```

Comment:
Similar to the above.

```suggestion
				if ( empty( $script->extra['after'] ) || ! is_array( $script->extra['after'] )  ) {
					continue;
				}
				$inline_size = 0;
				
```

Reply:
this doesn't make sense, it will break functionality. If empty extra['after'] $inline_size will be 0 and that's it, we don't need to skip the iteration with continue.